README.md

```diff
@@ -98,6 +98,21 @@ Objects are passed directly to Prettier as [options](https://prettier.io/docs/en
 
 NB: This option will merge and override any config set with `.prettierrc` files (for Prettier < 1.7.0, config files are ignored)
 
+### Additional Options
+
+- fileInfoOptions: Options that are passed to [prettier.getFileInfo](https://prettier.io/docs/en/api.html#prettiergetfileinfofilepath--options) in order to decide if a particular file needs to be formatted. As an example, this can be used to override the default ignorePath inferred from the current working directory.
+
+```json
+{
+  "rules": {
+    "prettier/prettier": [
+      true,
+      {"fileInfoOptions": {"ignorePath": "../../.prettierignore"}}
```

Comment:
Small note - ESLint has a third "slot" for options, I did not find a similar pattern for Stylelint, but I may just be not as familiar.

Theirs looks like this:
```
"prettier/prettier": ["error", { /* this is where Stylelint configures fileInfoOptions instead */ }, {
  "fileInfoOptions": {
    "withNodeModules": true
  }
}]
```

Let me know if there is a preferred approach instead.